cmd/builder/.goreleaser.yml

```diff
@@ -141,4 +138,8 @@ sboms:
     artifacts: archive
   - id: package
     artifacts: package
-    
+nightly:
+  version_template: '{{ incpatch .Version}}-nightly.{{ .Now.Format "200601021504" }}'
+  tag_name: nightly-builder
+  publish_release: true
```

Comment:
I wonder if this is going to be too spammy for people and we should set it to 'false'. Would we lose any benefits if we set this to false?

Reply:
Oki, I think drafts would be ideal if we can get the assets, since only people with write permissions would be able to check them

Reply:
i tried disabling `publish_release` but then no releases are created at all. Also no drafts unfortunately :(